### DISCUSSION TITLE
Database framework for Flet

### DISCUSSION THREAD

**Discussion by rizal-mujahiddan69** [flet-dev/flet#647] [Q&A] [answered]
I have question, how to flet can intergration with database and what the best database management tools for flet?

**Comment by ndonkoHenri** [comment, marked as answer]
Consider Flet as just the UI builder(frontend).
You just have to build up your backend consisting of all your python stuffs(sqlite3, mysql, ..etc) and find a way to link it to flet. (ex: retrieve some values in your database using python, and display it (add to page) using Flet `Text` control)
#pythonEverywhere

**Reply by Elisa734** [reply]
Bonsoir svp comment creer le backend

**Comment by goajos** [comment]
https://github.com/flet-dev/flet/discussions/4937

here is a very simple example of how you can use SQLAlchemy with Flet for Android to use the local SQLite storage. You can expand it but the idea stays the same.

**Reply by Elisa734** [reply]
Ok merci mais je ne sais pas trop comment l'utiliser dans mon projet

**Reply by Elisa734** [reply]
Comment faire appel a une base de donnee dans un code python avce flet
